CsCore/PlainNetClassLib/src/Plugins/CsCore/libs/StbImageWriteSharp/CRuntime.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace StbImageWriteSharp
```

Comment:
Library looks good in general, I think it makes sense to add it as a new dependency
Please also add the readme.md from https://github.com/StbSharp/StbImageWriteSharp